ogc-rs/src/console.rs

```diff
@@ -1,7 +1,3 @@
-//! The ``console`` module of ``ogc-rs``.
-//!
-//! This module implements a safe wrapper around the console functions.
-
```

Comment:
Any reason to delete the simple documentation headers of the files?

Reply:
I don't remember why I removed them earlier, added them back again.